### DISCUSSION TITLE
moments: Error 1093: You can't specify target table 'albums' for update in FROM clause (remove duplicates)

### DISCUSSION THREAD

**Discussion by droopified** [photoprism/photoprism#2758] [Q&A]
moments: Error 1093: You can't specify target table 'albums' for update in FROM clause (remove duplicates)

what does this mean? i get this error a lot but i check the directory and it doesnt contain any duplicate filenames.

**Comment by lastzero** [comment]
We have this bug report template so we don't have to ask you at 3am what database you are using and if you have read the system requirements in our setup guide yet?

👉 https://docs.photoprism.app/getting-started/#databases

**Comment by droopified** [comment]
so i have to uninstall mysql and install mariadb? been using mysql for years ill have to take time to figure out how to transfer everything

**Reply by lastzero** [reply]
You can also run both at the same time. What you do is ultimately up to you. We just don't have the time and energy to deal with all the problems MySQL has when only a few people actually use it.

**Reply by droopified** [reply]
would it be possible to export the photoprism database in sql format and upload that to a new mariadb setup and re-run the rescan or should i just start fresh? just wondering if it would be faster to get that database and run it again with most of the data and it fixing those errors.

**Reply by lastzero** [reply]
Really depends on the number of pictures and hardware you have vs the amount of time needed to export and reimport the SQL dump. If possible, start with a fresh database as it's easier.
